molecule/cookiecutter/scenario/driver/linode/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -0,0 +1,80 @@
+---
+{% raw -%}
+- name: Create
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  no_log: "{{ not (lookup('env', 'MOLECULE_DEBUG') | bool or molecule_yml.provisioner.log|default(false) | bool) }}"
+  vars:
+    ssh_user: root
+    ssh_port: 22
+    ssh_path: "{{ lookup('env', 'MOLECULE_EPHEMERAL_DIRECTORY') }}/ssh_key"
+    ssh_pass: "{{ lookup('password', lookup('env', 'MOLECULE_EPHEMERAL_DIRECTORY') + '/ssh_pass') }}"
+  tasks:
+    - name: Create SSH key
+      user:
+        name: "{{ lookup('env', 'USER') }}"
+        generate_ssh_key: true
+        ssh_key_file: "{{ ssh_path }}"
+      register: generated_ssh_key
+
+    - name: Create molecule instance(s)
+      linode:
+        name: "{{ item.name }}"
+        api_key: "{{ lookup('env', 'LINODE_API_KEY') }}"
+        datacenter: "{{ item.datacenter }}"
+        plan: "{{ item.plan }}"
+        distribution: "{{ item.distribution }}"
+        ssh_pub_key: "{{ generated_ssh_key.ssh_public_key }}"
+        password: "{{ ssh_pass }}"
+        state: present
+      register: server
+      with_items: "{{ molecule_yml.platforms }}"
+      async: 7200
+      poll: 0
+
+    - name: Wait for instance(s) creation to complete
+      async_status:
+        jid: "{{ item.ansible_job_id }}"
+      register: linode_jobs
+      until: linode_jobs.finished
+      retries: 300
+      with_items: "{{ server.results }}"
+
+    # Mandatory configuration for Molecule to function.
+
+    - name: Populate instance config dict
+      set_fact:
+        instance_conf_dict: {
```

Comment:
AFAICS curly bracket JSON-style mapping isn't necessary. YAML-way would be better readable.

Reply:
(but this can be fixed in follows-up PRs)

Reply:
This follows the style of all other drivers. Agree can be improved. 

Leaving it off for later style changes.